heroic-core/src/main/java/com/spotify/heroic/metric/MetricManagerModule.java

```diff
@@ -88,6 +90,16 @@
      */
     private final boolean failOnLimits;
 
+    /**
+     * If {@code true}, will enable logging of queries
+     */
+    private final boolean logQueries;
```

Comment:
This should be a `Feature` flag. It's already supported in the query path. (lookup the `Feature` class and its usages).